### ISSUE TITLE
Implement Read operations for types and instances

### ISSUE THREAD

**Issue by dmoranj** [telefonicaid/lwm2m-node-lib#30]
Currently, just the Resource Read operation is implemented. Implement also the Read operations for instances and object types.

**Comment by buzzware** [comment]
Yes it seems "read /3/0/1" is supported but not "read /3/0" - I get ResourceNotFound.
I suggest renaming this issue to something like "Implement Read for unspecified/all resource ids of type and instance"
